ISLE/isleapp.cpp

```diff
@@ -893,9 +979,10 @@ bool IsleApp::LoadConfig()
 	m_wideViewAngle = iniparser_getboolean(dict, "isle:Wide View Angle", m_wideViewAngle);
 	m_use3dSound = iniparser_getboolean(dict, "isle:3DSound", m_use3dSound);
 	m_useMusic = iniparser_getboolean(dict, "isle:Music", m_useMusic);
-	m_useJoystick = iniparser_getboolean(dict, "isle:UseJoystick", m_useJoystick);
-	m_joystickIndex = iniparser_getint(dict, "isle:JoystickIndex", m_joystickIndex);
+	m_useGamepad = iniparser_getboolean(dict, "isle:UseJoystick", m_useGamepad);
```

Comment:
Update the string; also I think we need to convey the clarification that Gamepad implies Joystick to the end user

Reply:
it will break old configs though...

Reply:
Good point. It's an internal value anyways.